src/node/internal/internal_zlib.ts

```diff
@@ -13,7 +13,122 @@ function crc32(data: ArrayBufferView | string, value: number = 0): number {
   validateUint32(value, 'value');
   return zlibUtil.crc32(data, value);
 }
+const constants = {
```

Comment:
If you have explicit testing you can make JS implementation a lot simpler

```suggestion
const constants = Object.keys().filter(k => k.startsWith())
```

Reply:
I don't see how that code would produce an object, but I think you're suggesting I select all the constants beginning with `Z_`, `BROTLI_`, etc. at runtime?

Reply:
I was implying something like this Object.keys().filter().reduce(fn, {})